package.json

```diff
@@ -88,14 +88,14 @@
     "enzyme-adapter-react-16": "1.15.2",
     "eslint": "6.8.0",
     "eslint-config-prettier": "6.9.0",
-    "eslint-plugin-jest": "23.2.0",
+    "eslint-plugin-jest": "23.3.0",
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.17.0",
     "eslint-plugin-react-hooks": "2.3.0",
     "fake-indexeddb": "3.0.0",
     "hard-source-webpack-plugin": "0.13.1",
     "html-webpack-plugin": "3.2.0",
-    "husky": "3.1.0",
+    "husky": "4.0.1",
```

Comment:
> Fix: make Yarn silent. When Husky was installed with Yarn 1.21.1, hooks output was too verbose (#626).
Support Yarn v2 new Plug'n'Play feature (thanks to @arcanis)
Improve path handling on Windows (in particular with Cygwin and Cmdr)
Remove dependencies and simplify code
Breaking requires Node 10+

Looks great.